docs/source/explore/get-started/install.rst

```diff
@@ -52,15 +52,61 @@ In addition, your network must have enough internet bandwidth to run Vircadia:
 .. note:: Running Vircadia on a high resolution display or in VR mode will need a much better graphics adapter, depending on your display's resolution and targeted frame rate.
 
 -----------------------------------
-Download Vircadia Installer
+Download Vircadia
 -----------------------------------
 
-You can download Vircadia through our website:
+* Windows |Windows|
 
-* `Download the Client Installer <https://vircadia.com/download-vircadia/#interface>`_
+    * `Windows 64-bit - Demeter v2020.3.0 Installer <https://cdn.vircadia.com/dist/release/Vircadia-Alpha-2020.3.1-Demeter.exe>`__ |DownloadWI|
+    * `Windows 64-bit Launcher (Experimental) <https://cdn.vircadia.com/dist/launcher/VircadiaLauncherSetup.exe>`__ |DownloadWL|
 
+* macOS |MacOS|
+
+    * `macOS Intel - Demeter v2020.3.0 Installer <https://cdn.vircadia.com/dist/release-experimental/Vircadia-Alpha-Interface-2020.3.1-Demeter.dmg>`__ |DownloadM|
+
+* Linux |Ubuntu|
```

Comment:
```suggestion
* Linux - Community Compiled |Ubuntu|
```

Reply:
I am not going to take this suggesting. At least not if no one like @daleglass or @ctrlaltdavid is for this as well.
It gives of the impression that Linux gets third grade support (if any at all), which in turn makes mine and Dale's efforts for Linux support seem less valued.

Reply:
I think if it's confusing then rescinding this and committing the below suggestion would be clearer.